exchanges/btcmarkets/btcmarkets.go

```diff
@@ -31,7 +31,7 @@ const (
 	btcMarketsAPIVersion = "/v3"
 
 	// UnAuthenticated EPs
-	btcMarketsAllMarkets         = "/markets/"
+	btcMarketsAllMarkets         = "/markets"
 	btcMarketsGetTicker          = "/ticker/"
```

Comment:
```suggestion
	btcMarketsGetTicker          = "/ticker"
```

Can also rid this trailing stash as it's not required even though it works

Reply:
👍